docs/configuration-reference.md

```diff
@@ -9,7 +9,6 @@ The Accelerated Bridge CNI configures networks through a CNI spec configuration
 * `ipam` (dictionary, optional): IPAM configuration to be used for this network.
 * `deviceID` (string, required): A valid pci address of a SWITCHDEV NIC's VF. e.g. "0000:03:02.3".
 * `vlan` (int, optional): VLAN ID to assign for the VF. Value must be in the range 0-4094 (0 for disabled, 1-4094 for valid VLAN IDs).
-* `mac` (string, optional): MAC address to assign for the VF
```

Comment:
we keep it for all cni version that don't support MAC as  runtime option

Reply:
Got it. Fixed